### PR TITLE
(#862) Fill out the Reports page

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

Add in some details about the Reports pages in CCM.

Note: This still requires some images to be added. I will add these to our tracking for when we do the images pass.

## Motivation and Context

Filling out some of the missing info in docs.

## Testing

* [ ] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [ ] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [x] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

- #862

### CODE REVIEW

File: input/en-us/central-management/usage/website/reports.md [hunk 1]
```diff
@@ -8,9 +8,37 @@ RedirectFrom:
...
+
+![Outdated Software Details view](/assets/images/ccm/reports/outdated-software-details.png)
+
+The Outdated Software report currently being viewed can be exported from this screen via the **Export** menu at the top-right.
+
+![Outdated Software Details view, showing the Export button options](/assets/images/ccm/reports/outdated-software-details-export.png)
+
+## Deployments
+
+The Deployment Plans report displays all completed Deployment Plans, and allows you to search for a subset of DeploymentPlans or adjust the sorting order, or filter the displayed reports by their final Status.
```
Comment (corbob) [inline input/en-us/central-management/usage/website/reports.md:37]: This feels a little clunky. I'm not sure if a semi-colon will help, or if perhaps it should be separated into two sentences?

```suggestion
The Deployment Plans report displays all completed Deployment Plans; it allows you to search for a subset of DeploymentPlans, adjust the sorting order, or filter the displayed reports by their final Status.
```

**Review by corbob — CHANGES_REQUESTED** [review]

**Comment by gep13** [comment]
@vexx32 can you update the commit in this PR to reference the #862 issue? That is where we are tracking all this work.

File: input/en-us/central-management/usage/website/reports.md (hunk 1 above)
Comment (vexx32) [reply input/en-us/central-management/usage/website/reports.md]: I opted for splitting the sentence here, thank you!

**Comment by vexx32** [comment]
@gep13 @corbob I think I've addressed both of those, thank you for looking it over! :3

**Comment by AdmiringWorm** [comment]
@corbob or @gep13, can you have another look over to ensure that the changes you wanted have been added?

**Comment by gep13** [comment]
@AdmiringWorm I have some local changes to this PR that have been made based on a conversation with @pauby I will be pushing up those changes shortly.

**Comment by gep13** [comment]
@corbob I have pushed up the changes that I had. @corbob I will leave this with you for final review.

**Review by corbob — APPROVED** [review]

**Pull request merged**